Src/AutoFoq/AutoFoqCustomization.fs

```diff
@@ -18,7 +18,8 @@ type AutoFoqCustomization(relay: ISpecimenBuilder) =
         AutoFoqCustomization(
             FilteringSpecimenBuilder(
                 MethodInvoker(
-                    FoqMethodQuery()),
+                    FoqMethodQuery(
+                        Fixture())),

```

Comment:
Done in [3702089](https://github.com/AutoFixture/AutoFixture/commit/3702089b1bd10cfd5c2d7221949636a01e367cc0).